index.html

```diff
@@ -2188,9 +2188,12 @@
             <code>"NotSupportedError"</code> , and abort these steps.
           </li>
           <li>
-            Otherwise, let <var>watchId</var> be a locally unique number
-            assigned to <var>options</var> and <var>callback</var> and store
-            them together as an <a>NFC watch</a>.
+            Let <var id="watch-suspended">suspended</var> be <code>false</code>.
```

Comment:
Thanks, this is a zombie from an earlier attempt. It was not the same 'suspended' as in the adapter, but it doesn't matter any more :).